samples/sensor/grow_r502a/boards/esp32.overlay

```diff
@@ -0,0 +1,23 @@
+&pinctrl {
+	uart2_default: uart2_default {
+		group1 {
+			pinmux = <UART2_TX_GPIO25>;
+		};
+		group2 {
+			pinmux = <UART2_RX_GPIO26>;
+			bias-pull-up;
+		};
+	};
+};
+
+&uart2 {
+	status = "okay";
+	current-speed = <57600>;
+	pinctrl-0 = <&uart2_default>;
+	pinctrl-names = "default";
+
+	grow_r502a {
+		compatible = "hzgrow,r502a";
+		int-gpios = <&gpio0 27 GPIO_ACTIVE_LOW>;
+	};
```

Comment:
this is effectively using uart as a bus, see my comment above.

Reply:
yes, fixed